mobile-app/app/screens/Main.tsx

```diff
@@ -2,10 +2,31 @@ import { StatusBar } from "expo-status-bar";
 
 import { SafeAreaProvider } from "react-native-safe-area-context";
 import { useThemeContext } from "@waveshq/walletkit-ui";
+import { useEffect } from "react";
+import { WalletAlert } from "@components/WalletAlert";
+import { translate } from "@translations";
 import { RootNavigator } from "./RootNavigator";
 
 export function Main(): JSX.Element {
   const { isLight } = useThemeContext();
+  useEffect(() => {
+    WalletAlert({
```

Comment:
Lets not add this alert in main component, I think lets create one separate component for this and include that component in portfolio page. So that we can reuse it for future alerts too